doc/modules/compose.rst

```diff
@@ -172,16 +179,16 @@ You can also provide custom feature names for the input data using
 
  * :ref:`composite_grid_search`
 
+|details-end|
 
-Notes
------
+.. note::
 
-Calling ``fit`` on the pipeline is the same as calling ``fit`` on
-each estimator in turn, ``transform`` the input and pass it on to the next step.
-The pipeline has all the methods that the last estimator in the pipeline has,
-i.e. if the last estimator is a classifier, the :class:`Pipeline` can be used
-as a classifier. If the last estimator is a transformer, again, so is the
-pipeline.
+    Calling ``fit`` on the pipeline is the same as calling ``fit`` on
+    each estimator in turn, ``transform`` the input and pass it on to the next step.
+    The pipeline has all the methods that the last estimator in the pipeline has,
+    i.e. if the last estimator is a classifier, the :class:`Pipeline` can be used
+    as a classifier. If the last estimator is a transformer, again, so is the
+    pipeline.
```

Comment:
Can you please move this note to the end of Section 6.1.1? Right after the paragraph
> All estimators in a pipeline, except the last one ...

Reply:
> Can you please move this note to the end of Section 6.1.1? Right after the paragraph
> 
> > All estimators in a pipeline, except the last one ...

Sure, that makes sense @ArturoAmorQ!

I moved the notes and updated the PR (see rendered docs [here](https://output.circle-artifacts.com/output/job/61004f36-bc62-48aa-9466-ef3d471e786b/artifacts/0/doc/modules/compose.html)).

Cheers!